### PR TITLE
Minor: fix the unused arguments used by formatting string

### PR DESCRIPTION
**Pull request by chia7712**
Both arguments are string type so the specifier should be ```%s```.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Review by rondagostino — COMMENTED** [review]
LGTM, thanks for the PR

**Review by rondagostino — COMMENTED** [review]
LGTM

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java [hunk 1]
```diff
@@ -537,7 +537,7 @@ private WorkerTask buildWorkerTask(ClusterConfigState configState,
                                       valueConverter, headerConverter, transformationChain, consumer, loader, time,
                                       retryWithToleranceOperator, herder.statusBackingStore());
         } else {
-            log.error("Tasks must be a subclass of either SourceTask or SinkTask", task);
+            log.error("Tasks must be a subclass of either SourceTask or SinkTask, and current is {}", task);
```
Comment (ijuma) [inline connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java]: Nit: I don't think you need the `and`

File: connect/runtime/src/main/java/org/apache/kafka/connect/util/TopicAdmin.java [hunk 2]
```diff
@@ -236,21 +236,21 @@ public boolean createTopic(NewTopic topic) {
...
+                    log.debug("Unable to create topic(s) '{}' since the brokers at {} do not support the CreateTopics API." +
                             " Falling back to assume topic(s) exist or will be auto-created by the broker.",
                             topicNameList, bootstrapServers);
                     return Collections.emptySet();
                 }
                 if (cause instanceof ClusterAuthorizationException) {
                     log.debug("Not authorized to create topic(s) '{}'." +
                             " Falling back to assume topic(s) exist or will be auto-created by the broker.",
-                            topicNameList, bootstrapServers);
+                            topicNameList);
```
Comment (ijuma) [inline connect/runtime/src/main/java/org/apache/kafka/connect/util/TopicAdmin.java]: Should we include the brokers instead of removing them? Same below.

**Review by ijuma — COMMENTED** [review]
Thanks for the PR. A couple of comments below.

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java (hunk 1 above)
Comment (chia7712) [reply connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java]: Got it :)

File: connect/runtime/src/main/java/org/apache/kafka/connect/util/TopicAdmin.java (hunk 2 above)
Comment (chia7712) [reply connect/runtime/src/main/java/org/apache/kafka/connect/util/TopicAdmin.java]: sure

**Review by ijuma — APPROVED** [review]
I tweaked the messages a bit, LGTM now.

**Comment by ijuma** [comment]
@chia7712 does this look OK to you?

**Review by andrewchoi5 — APPROVED** [review]

**Comment by chia7712** [comment]
> does this look OK to you?

LGTM :)

**Comment by ijuma** [comment]
retest this please

**Comment by chia7712** [comment]
retest this please

**Review by kkonstantine — APPROVED** [review]
The final version of this PR had a successful run in both jdk8 and jdk11. 
Merging since there are no conflicts since then. 

LGTM

**Comment by kkonstantine** [comment]
Merged to `trunk`, `2.5` and `2.4`

**Pull request merged**